src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/debian/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -11,6 +11,7 @@
   </PropertyGroup>
   
   <ItemGroup>
+    <NativeSplittableBinary Include="$(BinDir)libclrjit.so"/>
```

Comment:
@janvorli clrjit binary is still generated since we always do the standalone build of the JIT. I have simply included it to be in the nuget package so that we have the right things inplace by default.